composer.json

```diff
@@ -225,12 +227,11 @@
         "npm-asset/tablesaw": "~3.1.0",
         "oomphinc/composer-installers-extender": "~1.0 || ~2.0",
         "spatie/color": "^1.2",
-        "swiftmailer/swiftmailer": "6.2.4",
```

Comment:
We can't just remove this see #2705, if this is necessary it unfortunately can't make it in a minor release.

Reply:
Check. That seems it might/will run a different version as it's not locked, have we tested if that is compatible with our work? Also bear in mind these are things we need to write down in the PR so people are aware this version changes